index.html

```diff
@@ -63,6 +93,76 @@ <h2 class="text-primary">Add a new Book</h2>
         </div>
       </form>
     </div>
+
+    <div class="my-4 max-width-800 visually-hidden" id="contact">
+      <h2 class="text-primary text-center">Contact Information</h2>
+      <p class="text-center">
+        Do you have any questions or you just want to say "Hello"?<br />
+        You can reach out to any of us!
+      </p>
+      <ul class="list-group">
+        <li
+          class="
+            list-group-item
+            d-flex
+            justify-content-between
+            align-items-center
+          "
+        >
+          KARANGWA NTWALI JOSUE
+          <span class="badge rounded-pill">
+            <a
+              class="
+                btn
+                list-group-item list-group-item-action list-group-item-primary
+              "
+              href="https://github.com/Ntwali-Josue"
+              >GITHUB</a
+            >
+          </span>
+        </li>
+        <li
+          class="
+            list-group-item
+            d-flex
+            justify-content-between
+            align-items-center
+          "
+        >
+          DANIEL SAMUEL
+          <span class="badge rounded-pill"
+            ><a
+              class="
+                btn
+                list-group-item list-group-item-action list-group-item-primary
+              "
+              href="https://github.com/DanSam5k"
+              >GITHUB</a
+            >
+          </span>
+        </li>
+        <li
+          class="
+            list-group-item
+            d-flex
+            justify-content-between
+            align-items-center
+          "
+        >
+          ZAIN SADAQAT
+          <span class="badge rounded-pill">
+            <a
+              class="
+                btn
+                list-group-item list-group-item-action list-group-item-primary
+              "
+              href="https://github.com/zainsadaqat"
+              >GITHUB</a
+            >
+          </span>
+        </li>
+      </ul>
+    </div>
```

Comment:
- [ ] Your design pages does not include a `footer` as can be seen in the [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md) design. Kindly ensure to add the `footer` section to each page of your design.

**The wireframe template (footer section):**
![image](https://user-images.githubusercontent.com/12745474/129296302-65351dd6-bddc-4e52-8272-262b721a8438.png)